snowflake_blueprints/upload_file.py

```diff
@@ -93,37 +273,32 @@ def upload_data(source_full_path, table_name, insert_method, db_connection):
                 db_connection.execute(f'DROP TABLE {table_name}')
             except BaseException:
                 pass
-            sys.exit(EXIT_CODE_INVALID_WAREHOUSE)
+            sys.exit(errors.EXIT_CODE_INVALID_WAREHOUSE)
         if 'session does not have a current database' in str(db_e):
             print(
                 f'The database provided either does not exist or your user does not have access to it. If no database was provided, this user does not have a default warehouse.')
             print(db_e.orig)  # Avoids printing data to console
-            sys.exit(EXIT_CODE_INVALID_DATABASE)
+            sys.exit(errors.EXIT_CODE_INVALID_DATABASE)
         if 'is not recognized' in str(db_e):
             print(
                 'One or more of the values in your file does not match data_type of its column.')
             print(db_e.orig)  # Avoids printing data to console
-            sys.exit(EXIT_CODE_INVALID_UPLOAD_VALUE)
+            sys.exit(errors.EXIT_CODE_INVALID_UPLOAD_VALUE)
         if 'This session does not have a current schema.' in str(db_e):
```

Comment:
Same question about ifs using the string in operator as opposed to something more robust.

Reply:
Not that I'm aware of.